ansible/roles/rsyslog/tasks/main.yml

```diff
@@ -58,6 +58,15 @@
   changed_when: rsyslog__register_file_permissions.stdout|d()
   notify: [ 'Restart rsyslogd' ]
 
+- name: Create systemd-tmpfiles override
+  copy:
+    dest: '/etc/tmpfiles.d/rsyslog-var-log.conf'
+    owner: 'root'
+    group: 'root'
+    mode: '0755'
+    content: 'z {{ rsyslog__home }} 0775 root {{ rsyslog__group }} -'
+  when: rsyslog__unprivileged|bool and ansible_distribution == "Debian"
```

Comment:
Could you add a condition to check if the `systemd` is used on the host? Otherwise `/etc/tmpfiles.d/` directory might not be present and the role run will fail. For example:
```
when: ansible_service_mgr == "systemd" and ...
```

Reply:
Done as suggested. Did not know that non systemd systems are supported by debops.

Reply:
Thanks! Well, perhaps not fully supported, but I'm not enforcing `systemd`-only policy here. If there are users that want to use DebOps with Devuan, for example, and they provide PRs to improve support for it, I don't mind as long as it doesn't interfere with officially supported OSes (Debian, Ubuntu).